tests/relational_tests.rs

```diff
@@ -747,6 +748,82 @@ pub async fn linked() -> Result<(), DbErr> {
         }]
     );
 
+    let select_baker_with_customer = Baker::find().find_with_linked(baker::BakedForCustomer);
+
+    assert_eq!(
+        select_baker_with_customer
+            .build(sea_orm::DatabaseBackend::MySql)
+            .to_string(),
+        [
+            // FIXME: This might be faulty!
+            "SELECT `baker`.`id` AS `A_id`,",
+            "`baker`.`name` AS `A_name`,",
+            "`baker`.`contact_details` AS `A_contact_details`,",
+            "`baker`.`bakery_id` AS `A_bakery_id`,",
+            "`r4`.`id` AS `B_id`,",
+            "`r4`.`name` AS `B_name`,",
+            "`r4`.`notes` AS `B_notes`",
+            "FROM `baker`",
+            "INNER JOIN `cakes_bakers` AS `r0` ON `r0`.`baker_id` = `baker`.`id`",
+            "INNER JOIN `cake` AS `r1` ON `r1`.`id` = `r0`.`cake_id`",
+            "INNER JOIN `lineitem` AS `r2` ON `r2`.`cake_id` = `r1`.`id`",
+            "INNER JOIN `order` AS `r3` ON `r3`.`id` = `r2`.`order_id`",
+            "INNER JOIN `customer` AS `r4` ON `r4`.`id` = `r3`.`customer_id`",
```

Comment:
Should this be a inner join or a left join?
I found that other find_with/find_also functions use left join instead of inner join

Reply:
Yes, it should be `LEFT JOIN`